gnucash/report/trep-engine.scm

```diff
@@ -983,7 +985,16 @@ be excluded from periodic reporting.")
       gnc:pagename-display (N_ "Sign Reverses")
       "m1" (G_ "Reverse amount display for certain account types.")
       "global"
-      (keylist->vectorlist sign-reverse-list)))
+      (keylist->vectorlist sign-reverse-list))
+
+    (gnc-register-multichoice-option options
+      gnc:pagename-display optname-running-totals
+      "o2" (G_ "Display running totals as per report sort order?")
+      "none"
+      (list (vector 'none  (G_ "None"))
+            (vector 'all   (G_ "Grand Total and Subtotals"))
+            (vector 'grand (G_ "Grand Total Only"))
+            (vector 'sub   (G_ "Subtotals Only")))))
```

Comment:
I like the ability to choose the totals with a single multichoice. But is there a use case for "Subtotals only"?

Reply:
The short answer it is the same use case when the grant total is not useful yet I want to see subtotals. That's already an option, so this one is just extended to the running totals as well. The long answer is when the report shows a series of accounts, each presented with their own primary subtotal (as per the sorting options), but the grant total of all account is not relevant because the various accounts are not part of a group -- Maybe it's a mix of asset and expense accounts or whatever else I have to review and I don't need nor want to see the grand total nor running grant total because it is neither relevant nor useful.